docs/resources/cdn_resource.md

```diff
@@ -115,7 +115,7 @@ Optional:
 - `cache_http_headers` (Block List, Max: 1) Legacy option. Use the response_headers_hiding_policy option instead. (see [below for nested schema](#nestedblock--options--cache_http_headers))
 - `cors` (Block List, Max: 1) CORS header support option adds the Access-Control-Allow-Origin header to responses from CDN servers. (see [below for nested schema](#nestedblock--options--cors))
 - `country_acl` (Block List, Max: 1) Country access policy enables control access to content for specified countries. (see [below for nested schema](#nestedblock--options--country_acl))
-- `disable_cache` (Block List, Max: 1) Option enables browser caching. When enabled, content caching is completely disabled. (see [below for nested schema](#nestedblock--options--disable_cache))
+- `disable_cache` (Block List, Max: 1) Legacy option. Use the edge_cache_settings option instead. (see [below for nested schema](#nestedblock--options--disable_cache))
```

Comment:
Hey @MarianaStrix , sorry, I didn't notice when I first looked at your pull request, but it seems you update docs manually. Please use the `make doc-generate` command to autogenerate docs, it will include all needed info.

And also please check and update examples if there are used deprecated options.

Reply:
> Hey @MarianaStrix , sorry, I didn't notice when I first looked at your pull request, but it seems you update docs manually. Please use the make doc-generate command to autogenerate docs, it will include all needed info and option will be marked as Deprecated in the docs
I used `make doc-generate`

Reply:
@MarianaStrix https://github.com/G-Core/terraform-provider-gcore?tab=readme-ov-file#updating-docs
please use tfplugindocs

Reply:
Have you used it after adding the "Deprecated" attribute to the schema? For me it generates the following:
```suggestion
- `disable_cache` (Block List, Max: 1, Deprecated) Legacy option. Use the edge_cache_settings option instead. (see [below for nested schema](#nestedblock--options--disable_cache))
```

Reply:
Yes, on this code I use it and I get the same docs

Reply:
I'll find out it

Reply:
Done, thanks